frame/nft/src/test/impls/transfer.rs

```diff
@@ -0,0 +1,302 @@
+use std::{
+	collections::{BTreeMap, BTreeSet},
+	iter,
+};
+
+use codec::Encode;
+use composable_tests_helpers::test::{
+	block::process_and_progress_blocks, helper::assert_last_event,
+};
+use composable_traits::financial_nft::{FinancialNftProvider, NftClass};
+use frame_support::{
+	assert_ok,
+	traits::tokens::nonfungibles::{Inspect, Transfer},
+};
+
+use crate::{
+	test::{
+		mint_nft_and_assert,
+		mock::{new_test_ext, Event, Test},
+		ALICE, BOB, CHARLIE,
+	},
+	AccountIdOf, Instance, NftInstanceId, OwnerInstances, Pallet,
+};
+
+#[test]
+fn simple() {
+	//! Tests a simple transfer between 2 accounts, with only 1 total NFT existing.
```

Comment:
wow, that works? nice. so i would pefer on test because it is possible)

Reply:
I just thought it looked a bit strange like this:

```rust
/// Tests a simple transfer between 2 accounts, with only 1 total NFT existing.
#[test]
fn simple() {
```

with the test attribute between the comment and the test (also weird if you put the attribute above the comment, just can't win :laughing:)

I'm not super opinionated either way though, I can use the regular outer comment :+1: